zboxcore/sdk/uploadworker.go

```diff
@@ -618,6 +619,219 @@ func (req *UploadRequest) processUpload(ctx context.Context, a *Allocation) {
 	return
 }
 
+func (req *UploadRequest) processStreamingUpload(ctx context.Context, a *Allocation) {
```

Comment:
@iamrz1 I can't find InProgress callback in this function. Based on 'InProgress' we showing UI on clients

Reply:
It's included in the UploadRequest receiver (req) just like normal upload method.
FYI this is the worker part, CLI calls an Allocation member function named UploadStream, which takes the callback function and sets it in a UploadRequest member variable.

Reply:
Okay